cmd/loopd/swapclient_server.go

```diff
@@ -75,6 +75,7 @@ func (s *swapClientServer) LoopOut(ctx context.Context,
 		MaxSwapRoutingFee:   btcutil.Amount(in.MaxSwapRoutingFee),
 		MaxSwapFee:          btcutil.Amount(in.MaxSwapFee),
 		SweepConfTarget:     sweepConfTarget,
+		MaxSwapWaitTimeSec:  in.MaxSwapWaitTimeSec,
```

Comment:
I wouldn't mind grouping some changes together in a single commit, but that's my personal preference.